docs/querying/sql.md

```diff
@@ -952,6 +952,8 @@ You can additionally request a header by setting "header" to true in your reques
 In this case, the first result returned will be a header. For the `csv`, `array`, and `arrayLines` formats, the header
 will be a list of column names. For the `object` and `objectLines` formats, the header will be an object where the
 keys are column names, and the values are null.
+If you specified `sqlQueryId` in the [connection context parameters](#connection-context),
+the SQL query identifier will be returned in the header labeled `X-Druid-SQL-Query-Id`.
```

Comment:
```suggestion
Druid returns the SQL query identifier in the `X-Druid-SQL-Query-Id` header.
```
nit: prefer active

Reply:
If you don't specified `sqlQueryId` in the connection context parameters, a sqlQueryId will be generated by Druid and return in the response's `X-Druid-SQL-Query-Id` header